src/components/Plants/index.tsx

```diff
@@ -1,17 +1,85 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { ServerOrganization } from 'src/types/Organization';
 import strings from 'src/strings';
 import TfMain from 'src/components/common/TfMain';
-import Title from 'src/components/common/Title';
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import { Select } from '@terraware/web-components';
+import { listPlantingSites } from 'src/api/tracking/tracking';
+import { PlantingSite } from 'src/api/types/tracking';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+import { useHistory, useParams } from 'react-router-dom';
+import { APP_PATHS } from 'src/constants';
+import useSnackbar from 'src/utils/useSnackbar';
 
 type PlantsDashboardProps = {
   organization: ServerOrganization;
 };
 
 export default function PlantsDashboard(props: PlantsDashboardProps): JSX.Element {
+  const { organization } = props;
+  const [selectedPlantingSite, setSelectedPlantingSite] = useState<PlantingSite>();
+  const [plantingSites, setPlantingSites] = useState<PlantingSite[]>([]);
+  const theme = useTheme();
+  const { isMobile } = useDeviceInfo();
+  const { plantingSiteId } = useParams<{ plantingSiteId: string }>();
+  const history = useHistory();
+  const [snackbar] = useState(useSnackbar());
+
+  const setActivePlantingSite = useCallback(
+    (site: PlantingSite | undefined) => {
+      if (site) {
+        history.push(APP_PATHS.PLANTING_SITE_DASHBOARD.replace(':plantingSiteId', site.id.toString()));
+      }
+    },
+    [history]
+  );
+
+  useEffect(() => {
+    const populatePlantingSites = async () => {
+      const serverResponse = await listPlantingSites(organization.id);
+      if (serverResponse.requestSucceeded) {
+        setPlantingSites(serverResponse.sites ?? []);
+      } else {
+        snackbar.toastError();
```

Comment:
you can do `snackbar.toastError(serverResponse.error)` if we want to show more details on the error